.github/workflows/cacert-publish.yml

```diff
@@ -1,7 +1,7 @@
 name: Publish ca-certificates
 on:
   push:
-    branches: [ master ]
+    branches: [ master, 'feature/**', 'bugfix/**' ]
```

Comment:
You are right, I was only looking for the build part here and ignored that it will be published in the end. As the build is covered by the `linux.yml` GH action as well I will remove it here.